### ISSUE TITLE
TLS call-home : Client session establishments with redundant <get-schema>

### ISSUE THREAD

**Issue by Bharanidharan-S** [CESNET/netopeer#200]
Dear Team,

We use TLS call-home feature with "netopeer2-server 1.1.77".
Could see <get-schema> involvement during each client session establishments, even we use nc_client_set_schema_searchpath("/usr/local/yang/").
Please note, all netopeer2-servers are loaded with the same yang files & in other separate machine, all netopeer clients are running or trying to establish more connections with different servers.
Its acceptable, if <get-schema> involvement exist during first client establishment. But the <get-schema> is invoked in all other client session establishments too, when the same yang files are already stored in /usr/local/yang during the first client establishment.

Can someone please let me know why the <get-schema> involved in all session establishments.

For more clarity, have provided connection details below:

                                                                                   - - - - - - - - - - - - - - - -
In client-side,                              |--------------->     |   netopeer2-server (1)  |
nc_callhome_accept(),                 |                              - - - - - - - - - - - - - - - -
 - - - - - - - - - - - - - - - -             |
|             client (1),           |  <-----|                             - - - - - - - - - - - - - - - - 
|             client (2),           |  <-----(ipv6 TLS)---->      |   netopeer2-server (2)  |
|             client (3),           |  <-----|                             - - - - - - - - - - - - - - - - 
 - - - - - - - - - - - - - - - -             |
                                                    |                              - - - - - - - - - - - - - - - - 
                                                    |--------------->     |   netopeer2-server (3)  |
                                                                                   - - - - - - - - - - - - - - - -

Thanks in advance..

**Comment by Bharanidharan-S** [comment]
Why get-schema, even if we use nc_client_set_schema_searchpath("/usr/local/yang/")?
We use different client server versions - does it bother?
- netopeer2-cli 2.0.64
- netopeer2-server 1.1.77

**Comment by michalvasko** [comment]
Yes, I think this was improved fairly recently so if you are still using netopeer2 v1, then it is certainly not fixed in the libnetconf2 you have installed with it. If you use the current versions, the local modules should be used when possible.
